types/storybook__react/index.d.ts

```diff
@@ -12,15 +13,21 @@ import * as React from 'react';
 export type Renderable = React.ComponentType | JSX.Element;
 export type RenderFunction = () => Renderable | Renderable[];
 
+export interface DecoratorParameters {
+    [key: string]: any;
+}
 export type StoryDecorator = (story: RenderFunction, context: { kind: string, story: string }) => Renderable | null;
 
 export interface Story {
     readonly kind: string;
-    add(storyName: string, callback: RenderFunction): this;
+    add(storyName: string, callback: RenderFunction, parameters?: DecoratorParameters): this;
     addDecorator(decorator: StoryDecorator): this;
+    addParameters(parameters: DecoratorParameters): this;
 }
 
 export function addDecorator(decorator: StoryDecorator): void;
+export function addParameters(parameters: DecoratorParameters): void;
```

Comment:
Here's the top-level `addParameters` function: https://github.com/storybooks/storybook/blob/next/lib/core/src/client/preview/client_api.js#L51-L53